src/lib/config/networkConfigType.ts

```diff
@@ -4,6 +4,14 @@ export interface BundlrConfig {
   providerUrl: string;
 }
 
+interface ArweaveConfig {
+  host: string;
+  port: number;
+  protocol: 'http' | 'https';
+  timeout: number;
+  logging: boolean;
+}
```

Comment:
There is a pre-exiting type called `ApiConfig` in arweave's library (`arweave/node/lib/api.d.ts`) that we should be able to use --

```
export interface ApiConfig {
    host?: string;
    protocol?: string;
    port?: string | number;
    timeout?: number;
    logging?: boolean;
    logger?: Function;
    network?: string;
}
```